server/src-lib/Hasura/GraphQL/Resolve/Action.hs

```diff
@@ -414,10 +421,11 @@ callWebhook manager outputType outputFields reqHeaders confHeaders
                pure (webhookResponse, mkSetCookieHeaders responseWreq)
 
          | HTTP.statusIsClientError responseStatus -> do
-             ActionWebhookErrorResponse message maybeCode <-
+             ActionWebhookErrorResponse message maybeCode maybeExtension <-
```

Comment:
We should not take `code` anymore.

Reply:
See spec: https://github.com/hasura/graphql-engine/issues/4001#issuecomment-608408505

It is find to do this "breaking" change because we introduced it in beta.

Reply:
I kept `code` thinking about the breaking change, I'll fix this

Reply:
@codingkarthik Cool. Also, could you also update the docs with this PR itself?

Reply:
Yes, I'll make the changes